### DISCUSSION TITLE
Invalid command 'OIDCStripCookies', perhaps misspelled or defined by a module

### DISCUSSION THREAD

**Discussion by vamshibea** [OpenIDC/mod_auth_openidc#613] [Q&A] [answered]
Using Apache 2.4.6 Version on Centos. Installed "mod_auth_openidc.so" using yum:
yum install mod_auth_openidc

 "mod_auth_openidc.so" is configured in apache:
LoadModule auth_openidc_module modules/mod_auth_openidc.so

Most of the Generic OIDC Parameters are working fine But when i add below "OIDCStripCookie", I am getting Exception saying "Invalid command 'OIDCStripCookies', perhaps misspelled or defined by a module"

OIDCStripCookies mod_auth_openidc_session mod_auth_openidc_state

**Comment by zandbelt** [comment, marked as answer]
upgrade to a more recent version please

**Reply by vamshibea** [reply]
Hi @zandbelt, Thanks for the Response. I will try to install more recent version of Openidc.

**Reply by vamshibea** [reply]
Installed Latest Version of Openidc. I am now able to to use OIDCStripCookies. Thanks @zandbelt
